### PR TITLE
Fixed accidence in (chair and techleads readme)

### PR DESCRIPTION
**Pull request by Krishnamohan-Yerrabilli**
### Why:

Found typos, incorrect article usage

### What's being changed:

Changed "Place" to "A place" -> missing a determiner before it fixed
Changed "Recogition" to "Recognition" -> typo fixed
Changed "commuity" to "community " -> typo fixed
Changed "list and slack" to "list and the slack" added a article 

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the self-review checklist.

### CODE REVIEW

**Comment by Krishnamohan-Yerrabilli** [comment]
@mrbobbytables Opened this PR with setting up new branch(Krishnamohan-Yerrabilli:new-branch-chair-TL-rm) I hope things are clear now to review, plz check it ty.

File: contributors/chairs-and-techleads/README.md [hunk 1]
```diff
@@ -85,8 +85,8 @@ Try #shoutouts on the Kubernetes slack!
       given the appropriate level of priority.
 #### Awards
 There are a few times a year that Kubernetes and the larger CNCF community 
-intentionally recognizes members of the commuity for certain roles and 
-accomplishments. The leads@ mailing list and slack channel will have 
+intentionally recognizes members of the community for certain roles and 
+accomplishments. The leads@ mailing list and the slack channel will have
```
Comment (jberkus) [inline contributors/chairs-and-techleads/README.md:89]:
```suggestion
accomplishments. The leads@ mailing list and slack channel will have 
```

The second "the" isn't desirable.  In this clause "leads@" modifies both "mailing list" and "slack channel", so putting a second "the" there breaks that pairing.

File: contributors/chairs-and-techleads/README.md [hunk 2]
```diff
@@ -112,4 +112,4 @@ announcements for when nominations are open.
 [CNCF Awards]: https://github.com/cncf/awards 
 [cncf swag store]: (https://store.cncf.io/collections/kubernetes) 
 [calendar template]: https://github.com/kubernetes/community/blob/master/communication/calendar-guidelines.md#calendar-event-template
-[leads@kubernetes.io]: https://github.com/kubernetes/k8s.io/blob/aa1214087863c7a850bf84936bd1d1b67d0064a9/groups/groups.yaml#L6
\ No newline at end of file
+[leads@kubernetes.io]: https://github.com/kubernetes/k8s.io/blob/aa1214087863c7a850bf84936bd1d1b67d0064a9/groups/groups.yaml#L6
```
Comment (jberkus) [inline contributors/chairs-and-techleads/README.md:115]: er, what happened here?  Whitespace?

**Review by jberkus — CHANGES_REQUESTED** [review]
Thanks for edits!  Please see my edits to your edits.  Thanks.

File: contributors/chairs-and-techleads/README.md (hunk 1 above)
Comment (Krishnamohan-Yerrabilli) [reply contributors/chairs-and-techleads/README.md]: suggestions applied

File: contributors/chairs-and-techleads/README.md (hunk 2 above)
Comment (Krishnamohan-Yerrabilli) [reply contributors/chairs-and-techleads/README.md:115]: yeah looks like, let me recheck

**Pull request merged**